### PR TITLE
dash/grouped-samples-demopage

### PR DESCRIPTION
**Pull request by sebastianbochan**
Assigned demos into two categories.

### CODE REVIEW

**Review by ddragula — APPROVED** [review]

File: samples/dashboards/demo/cloud-monitoring/demo.details [hunk 1]
```diff
@@ -6,7 +6,7 @@ js_wrap: b
 tags:
   - Highcharts Dashboards demo
 categories:
-  - General:
+  - Advanced:
         priority: 1
```
Comment (karolkolodziej) [inline samples/dashboards/demo/cloud-monitoring/demo.details:10]: Accounting already has the same priority. Won't it be an issue?

**Review by karolkolodziej — APPROVED** [review]
Looks good, I would only check the priorities with someone from the demo pages 😉

**Review by stitot — CHANGES_REQUESTED** [review]
Looks good, but a cleanup suggestion: let's add `priority` in each `demo.details` in both categories, in increments of 10. That way we have full control on order of demos, and we can add new ones were we want.

**Review by stitot — COMMENTED** [review]
IMO we should use increments of 10 in `priority`, meaning 10,20,30 etc. instead of 1,2,3 etc. That way we can inject new demos where we want without adjusting every single `priority` each time. Your call.

**Comment by sebastianbochan** [comment]
I'm not convinced, because of requests to add more categories, set different orders, etc. For me, it's only artificially flexibly.

File: samples/dashboards/demo/accounting/demo.details [hunk 2]
```diff
@@ -6,7 +6,7 @@ js_wrap: b
 tags:
   - Highcharts Dashboards demo
 categories:
-  - General:
+  - Advanced:
        priority: 1
```
Comment (stitot) [inline samples/dashboards/demo/accounting/demo.details:10]:
```suggestion
       priority: 10
```

File: samples/dashboards/demo/minimal/demo.details [hunk 3]
```diff
@@ -6,7 +6,7 @@ js_wrap: b
 tags:
   - Highcharts Dashboards demo
 categories:
-  - General:
+  - Basic:
         priority: 1
```
Comment (stitot) [inline samples/dashboards/demo/minimal/demo.details:10]:
```suggestion
        priority: 10
```

**Review by stitot — CHANGES_REQUESTED** [review]
Aaaaaaalmost there

**Review by stitot — APPROVED** [review]

**Pull request merged**